src/renderer/src/scenes/main-scene/MainScene.tsx

```diff
@@ -45,10 +76,39 @@ const Nav: Component = () => {
           <i class="ri-stack-fill nav__queue-icon" />
         </IconButton>
       </div>
+      {os() !== "darwin" && <NavWindowControls maximized={maximized} setMaximized={setMaximized} />}
     </nav>
   );
 };
 
+function NavWindowControls(props: { maximized: Accessor<boolean>; setMaximized: Setter<boolean> }) {
```

Comment:
`WindowControls` seems a better fit. But that is highly subjective. I will leave it up to you which name you will choose

Reply:
agreed, changed to WindowControls